Source/SwiftLintCoreMacros/RuleConfigurationMacros.swift

```diff
@@ -74,7 +74,9 @@ enum AutoConfigParser: MemberMacro {
                 """
                 for option in nonInlinedOptions {
                     """
-                    try \(raw: option).apply(configuration[$\(raw: option).key], ruleID: Parent.identifier)
+                    if configuration.keys.contains($\(raw: option).key) {
+                        try \(raw: option).apply(configuration[$\(raw: option).key], ruleID: Parent.identifier)
+                    }
```

Comment:
```suggestion
                    if let value = configuration[$\(raw: option).key] {
                        try \(raw: option).apply(value, ruleID: Parent.identifier)
                    }
```

Reply:
Shouldn't we support overwriting the default value with null here?

Reply:
Passing `nil` is a no-op right now and with the change this behavior will be retained.